src/main/java/edu/greenblitz/tobyDetermined/subsystems/swerve/SdsSwerveModule.java

```diff
@@ -254,11 +254,11 @@ public void setRotIdleModeCoast() {
 	}
 
 	public static class SdsSwerveModuleConfigObject {
```

Comment:
Like other cases, refactor config classes to their own file.
Also, I'm not sure which java are we using, but if we use java version >= 14, then lookup about 'record'

Reply:
in WPILib 2024 we should use Java 14. the current WPILib supports java 11